x-pack/plugins/rule_registry/server/rule_data_client/index.ts

```diff
@@ -116,29 +117,22 @@ export class RuleDataClient implements IRuleDataClient {
 
     const clusterClient = await this.getClusterClient();
 
-    const { body: aliasExists } = await clusterClient.indices.existsAlias({
-      name: alias,
-    });
-
     const concreteIndexName = `${alias}-000001`;
-
-    if (!aliasExists) {
```

Comment:
I removed this check so that we can add the `.alerts-security.alerts` alias to the `.siem-signals` indices *before* the `.alerts-security.alerts` concrete indices are actually created. This way we can treat all existing alerts as though they were migrated to `.alerts` and don't have to worry about handling old and new alerts separately.

Reply:
This check has now been replaced with a more specific check - querying to find concrete indices rather than the alias.